plugins/inputs/logfile/fdlimit/fdlimit_windows.go

```diff
@@ -0,0 +1,15 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build windows
+// +build windows
+
+package fdlimit
+
+//On Windows, Go uses the CreateFile API, which is limited to 16K files; therefore, non-changeable from within a running process
+//For more information on the API,please follow https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-createfilea
+const hardLimitFileDescriptor = 16384
```

Comment:
I skimmed the docs and I don't see where this value comes from. The value `16384` comes from `FILE_ATTRIBUTE_ENCRYPTED`

Reply:
The documents are for the API. For the discussion regarding the hard limit, my SQL has encountered the same issue https://bugs.mysql.com/bug.php?id=24509 and decided to go with 16384. Another example could be found with [go-ethereum](https://github.com/ethereum/go-ethereum/blob/8a134014b4b370b4a3632e32a2fc8e84ee2b6947/common/fdlimit/fdlimit_windows.go)